docs/openapi-fetch/testing.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Testing
+---
+
+# Testing
+
+Testing openapi-fetch is best done in a test runner that supports TypeScript, such as [Vitest](https://vitest.dev/) or Jest.
+
+## Mocking Requests
+
+To test requests, the `fetch` option can be supplied with any spy function like `vi.fn()` (Vitest) or `jest.fn()` (Jest).
```

Comment:
🤔 I’ll probably rewrite this to recommend [MSW](https://mswjs.io/) as I’ve had some good success with it, but will probably wait until I refactor this codebase to use it for testing. Treat this as more of a placeholder.